packages/core/helper-plugin/src/utils/getFetchClient/index.js

```diff
@@ -1,12 +1,23 @@
 import instance from '../fetchClient';
+import isAbsoluteUrl from '../isAbsoluteUrl';
+import addPrependingSlash from '../addPrependingSlash';
```

Comment:
I don't see any value in having these in separate files, I don't think they're used anywhere else are they? IMO let's just inline them in `checkUrl` function

Reply:
Ok, I understand thanks